exec/java-exec/src/main/java/org/apache/drill/exec/store/dfs/FileSystemPlugin.java

```diff
@@ -54,6 +54,9 @@
  * references to the FileSystem configuration and path management.
  */
 public class FileSystemPlugin extends AbstractStoragePlugin {
+  private static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(FileSystemPlugin.class);
+
+  public static final String FS_DEFAULT_NAME = "fs.default.name";
```

Comment:
It was mentioned that `fs.default.name` is deprecated. Can we replace it with `fs.defaultFS`?

Reply:
This was part of the magic in @chunhui-shi 's original fix. For some reason if a user defined **fs.default.name** in the hive storage plugin this was passed on to the DrillFileSystem. We even set *fs.defaultFS** on line 79, but if the deprecated **fs.default.name** option is set as well it's value wins.

After doing an RCA I will delete this.